app/models/fee_liability.rb

```diff
@@ -21,6 +21,10 @@ def paid?
     govpay_payment_status == 'success'
   end
 
+  def failed?
```

Comment:
Seems sensible.  I added it as a convenience method to be used when checking the status in #post_pay.